app/controllers/api/cloud_volumes_controller.rb

```diff
@@ -34,7 +34,8 @@ def delete_resource_main_action(_type, cloud_volume, _data)
 
     def options
       if (id = params["id"])
-        render_update_resource_options(id)
+        action = params["option_action"] || "update"
+        render_update_resource_options(id, action)
```

Comment:
Could we `send("render_#{action}_resource_options(id)")` instead?  Then we would localize the methods to the controller that has those actions